app/Http/Controllers/EmployeeController.php

```diff
@@ -21,12 +23,14 @@ public function show()
     public function create(EmployeeRequest $request)
     {
         // dd($request);
-        Employee::create([
+        $employee = Employee::create([
             'name' => $request->name,
             'role' => $request->role,
             'location' => $request->location,
             'salary' => $request->salary,
         ]);
+        // dd($employee);
+        Wallet::create(['employee_id' => $employee->id]);
```

Comment:
A function should only do what it's meant to do. Move this other code block to a new function and call it from here